android/quest/src/main/java/org/smartregister/fhircore/quest/data/report/measure/MeasureReportRepository.kt

```diff
@@ -61,4 +61,9 @@ constructor(
       registerId = measureReportConfiguration.registerId
     )
   }
+
+  /** returns start date of campaign to get the month/year list start month */
+  fun getCampaignStartDate(): String {
+    return measureReportConfiguration.registerDate
+  }
```

Comment:
View model should have access to the configuration model. I think it is not necessary to have this in the repository class.